webapp/i18n/en.json

```diff
@@ -818,6 +818,9 @@
   "admin.sql.keyTitle": "At Rest Encrypt Key:",
   "admin.sql.maxConnectionsDescription": "Maximum number of idle connections held open to the database.",
   "admin.sql.maxConnectionsExample": "E.g.: \"10\"",
+  "admin.sql.connectionTimeoutTitle": "Connection Timeout:",
+  "admin.sql.connectionTimeoutDescription": "The number of seconds a query can be executing for before being cancelled.",
```

Comment:
Ok, thanks.

The title should be "Query Timeout".
The help text should be: "The number of seconds to wait for a response from the database after opening a connection and sending the query. Errors that you see in the UI or in the logs as a result of a query timeout can vary depending on the type of query. This setting has no effect on PostgreSQL databases."

How long before the PostgreSQL driver bug is fixed? If it's going to be a long time then consider renaming to "MySQL Query Timeout" and removing the last sentence of the help text.